pkgs/servers/foundationdb/cmake.nix

```diff
@@ -1,178 +0,0 @@
-# This builder is for FoundationDB CMake build system.
-
-{
-  lib,
-  fetchFromGitHub,
-  cmake,
-  ninja,
-  python3,
-  openjdk,
-  mono,
-  pkg-config,
-  msgpack-cxx,
-  toml11,
-  jemalloc,
-  doctest,
-
-  gccStdenv,
-  llvmPackages,
-  useClang ? false,
```

Comment:
Can you explain why you removed this? It was done in the "move to by-name" commit with no further explanation.

Reply:
Bump.

Reply:
probably removed it by accident during cleanup. should we switch to clang as default stdenv?